pyproject.toml

```diff
@@ -184,6 +184,7 @@ pydocstyle.convention = "google"
   "S404", # subprocess calls are for tests
   "S603", # do not require `shell=True`
   "S607", # partial executable paths
+  "PLC2701", # Access to a protected member
```

Comment:
That's I would leave as is and refactor it later. Could you please revert changes of this commit?